content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Kops
+description: Instructions to setup Kops for use with Istio.
+weight: 20
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kops/
```

Comment:
If this is a new page, you probably don't need any aliases. They are used for when a page moves.